src/client/metric.rs

```diff
@@ -107,6 +99,35 @@ pub enum Space {
     EByte
 }
 
+impl Space {
+    fn from_u8(x: u8) -> Option<Self> {
+        match x {
+            0 => Some(Space::Byte),
```

Comment:
Can we have there numbers as constants? Ideally named the same as the `#define`s in `src/include/pcp/pmapi.h`?

See `src/include/pcp/pmapi.h:83`